pyproject.toml

```diff
@@ -8,40 +8,48 @@ issue_format = "[#{issue}](https://github.com/pulp/pulp-cli/issues/{issue})"
 start_string = "[//]: # (towncrier release notes start)\n"
 underlines = ["", "", ""]
 
-[[tool.towncrier.type]]
-directory = "feature"
-name = "Features"
-showcontent = true
+    [[tool.towncrier.section]]
+    path = ""
+    name = ""
```

Comment:
Why are these indented? Is the following towncrier types pulling double duty for both the cli & glue section?